Classes/Issues/Comments/Images/IssueCommentImageCell.swift

```diff
@@ -24,7 +24,8 @@ UIGestureRecognizerDelegate {
 
     weak var delegate: IssueCommentImageCellDelegate? = nil
     let imageView = UIImageView()
-
+    
+    private var imageUrl: URL!
```

Comment:
This shouldn't be implicit unwrap since `bindViewModel(...)` **must** be called first. Can we change it to optional and just `guard` around it?

Reply:
Done!